modules/exploits/osx/local/acronis_trueimage_xpc_privesc.rb

```diff
@@ -0,0 +1,162 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::File
+  include Msf::Post::Common
+  include Msf::Post::Process
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Acronis TrueImage XPC Privilege Escalation',
+        'Description' => %q{
+          Acronis TrueImage versions 2019 update 1 through 2021 update 1
+          are vulnerable to privilege escalation. The `com.acronis.trueimagehelper`
+          helper tool does not perform any validation on connecting clients,
+          which gives arbitrary clients the ability to execute functions provided
+          by the helper tool with `root` privileges.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Csaba Fitzl', # @theevilbit - Vulnerability Discovery
+          'Shelby Pace' # Metasploit Module and Objective-c code
+        ],
+        'Platform' => [ 'osx' ],
+        'Arch' => [ ARCH_X64 ],
```

Comment:
Is this for arm64 macOS?

Reply:
Looks like no ARM support for Acronis applications: https://kb.acronis.com/content/62562.